collect_stata/dataset.py

```diff
@@ -9,41 +17,37 @@ class Dataset:
     """
     Dataset allows the user to read, test and export data in different formats.
 
-    Example:
-
-        dataset = Dataset()
-        dataset.read_stata("../input/dataset.dta")
-        dataset.write_json("../output/dataset.json")
+    Args:
```

Comment:
Where do these args come from / belong to? There are not passed in the __init__ method.